src/Entities/MembershipApplication.php

```diff
@@ -953,6 +953,14 @@ public function isCancelled() {
 		return $this->status < 0 && abs( $this->status ) & abs( self::STATUS_CANCELED );
 	}
 
+	/**
+	 * @since 4.2
+	 * @return bool
+	 */
+	public function isDeleted() {
```

Comment:
this is a bit odd grammar wise. What about wasDeleted? I was about to submit a PR for this and then noticed we already have isCancelled as well. No strong opinion from my side on this